demo/video_gpuaccel_demo.py

```diff
@@ -32,15 +35,19 @@ def parse_args():
         '--device', default='cuda:0', help='Device used for inference')
     parser.add_argument(
         '--score-thr', type=float, default=0.3, help='Bbox score threshold')
-    parser.add_argument('--out', type=str, help='Output video file')
-    parser.add_argument('--show', action='store_true', help='Show video')
+    parser.add_argument(
+        '--show', action='store_true', help='Show the detection results')
```

Comment:
need to update:
docs/en/user_guides/inference.md
docs/zh_cn/user_guides/inference.md
docs/en/user_guides/inference.md
docs/zh_cn/user_guides/inference.md